software/ChromeAndPuppeteer.md

```diff
@@ -0,0 +1,135 @@
+# (Headless) Chrome and Puppeteer on Graviton
+
+“[Puppeteer](https://pptr.dev/) is a Node.js library which provides a high-level API to control Chrome/Chromium over the DevTools Protocol. 
+Puppeteer runs in headless mode by default, but can be configured to run in full ("headful") Chrome/Chromium."
+It can serve as a replacement for  Phantomjs.
+“PhantomJS (phantomjs.org) is a headless WebKit scriptable with JavaScript. The latest stable release is version 2.1.
+Important: PhantomJS development is suspended until further notice (see #15344 for more details).“
+The APIs are different, so code targetting PhantomJS has to be rewritten when moving to Puppeteer (see Appendix).
+Puppeteer has 466 contributors and 364k users.
+
+### Get a recent version of NodeJS.
+
+Ubuntu-22:
```

Comment:
Ubuntu 22 -> Ubuntu 22.04LTS

To limit confusion to the off chance someone files an issue that this doesn't work on 22.10. :)